src/validate.js

```diff
@@ -1659,6 +1659,18 @@ function F_VALIDATE(disabledHL) {
 		var reverseSpeed = segment.$revMaxSpeed;
 		var forwardSpeedUnverified = segment.$fwdMaxSpeedUnverified;
 		var reverseSpeedUnverified = segment.$fwdMaxSpeedUnverified;
+		var speedConversionFactor = 1;  // default to metric
+		var speedMultipleFactor = 10;   // default to limits being set in multiples of 10
```

Comment:
This should be configured in `i18n` repo as a default parameter for this check. Then modified in `i18n` for each country.